pymeasure/instruments/rohdeschwarz/sfm.py

```diff
@@ -0,0 +1,1501 @@
+
+        valid range: 0 .. 2047
+        """,
+        validator = strict_range,
+        values=[0, 2047],
+        cast=int
+        )
+
+    nicam_additional_bits = Instrument.control(
+        "SOUR:TEL:MOD:NIC:AUD:ADD?",
+        "SOUR:TEL:MOD:NIC:AUD:ADD %d",
+        """ A int property that controls the additional data in the NICAM  modulator
+
+        valid range: 0 .. 2047
+        """,
+        validator = strict_range,
+        values=[0, 2047],
+        )
+
+    nicam_control_bits = Instrument.control(
+        "SOUR:TEL:MOD:NIC:AUD:CONT?",
+        "SOUR:TEL:MOD:NIC:AUD:CONT %d",
+        """ A int property that controls the additional data in the NICAM  modulator
+
+        valid range: 0 .. 3
+        """,
+        validator = strict_range,
+        values=[0, 3],
+        )
+
+    nicam_bit_error_rate = Instrument.control(
+        "SOUR:TEL:MOD:NIC:BIT?",
+        "SOUR:TEL:MOD:NIC:BIT %g",
+        """ A float property that controls the artifical bit error rate.
+
+        valid range: 1.2E-7 .. 2E-3
+        """,
+        validator = strict_range,
+        values=[1.2E-7, 2E-3],
+        )
+
+    nicam_bit_error_enabled = Instrument.control(
+        "SOUR:TEL:MOD:NIC:BIT:STAT?",
+        "SOUR:TEL:MOD:NIC:BIT:STAT %d",
+        """ A bool property that controls the status of an artifical bit error rate to be applied,
+
+        ======  =======
+        Value   Meaning
+        ======  =======
+        False   artificial BER disabled
+        True    artificial BER enabled
+        ======  =======
+        """,
+        validator = strict_discrete_set,
+        values={False:0, True:1},
+        map_values = True,
+        )
+
+    nicam_carrier_frequency = Instrument.control(
+        "SOUR:TEL:MOD:NIC:CARR:FREQ?",
+        "SOUR:TEL:MOD:NIC:CARR:FREQ %g",
+        """ A float property that controls the frequency of the NICAM carrier
+
+        valid range: 33.05 MHz +/- 0.2 Mhz
+        """,
+        validator = strict_range,
+        values=[32.85E6, 33.25E6],
+        )
+
+    nicam_intercarrier_frequency = Instrument.control(
+        "SOUR:TEL:MOD:NIC:INT:FREQ?",
+        "SOUR:TEL:MOD:NIC:INT:FREQ %g",
+        """ A float property that controls the inter-carrier frequency of the NICAM carrier
+
+        valid range: 5 .. 9 MHz
+        """,
+        validator = strict_range,
+        values=[5E6, 9E6],
+        )
+
+    nicam_carrier_level = Instrument.control(
+        "SOUR:TEL:MOD:NIC:CARR:LEV?",
+        "SOUR:TEL:MOD:NIC:CARR:LEV %g",
+        """ A float property that controls the value of the NICAM carrier
+
+        valid range: -40 .. -13 dB
+        """,
+        validator = strict_range,
+        values=[-40, 13],
+        )
+
+    nicam_carrier_enabled = Instrument.control(
+        "SOUR:TEL:MOD:NIC:CARR:STAT?",
+        "SOUR:TEL:MOD:NIC:CARR:STAT %s",
+        """ A bool property that controls if the NICAM carrier is switched on or off
+
+        ======  =======
```

Comment:
I think if the name is clear, and already suffixed _enabled, you don't need the tables, and I'll stop pointing out every single one from here on. Please be so kind and check those yourself.